e2e/api/steps.js

```diff
@@ -55,8 +55,6 @@ module.exports = {
 
     await assertCallbackError('ServedDocuments', data[eventName].invalid.ServedDocuments.blankOtherDocuments,
       'CONTENT TBC: please enter a valid value for other documents');
-    await assertCallbackError('Date', data[eventName].invalid.Date.threeDaysBeforeToday,
```

Comment:
`threeDaysBeforeToday` entry could be removed from api test fixture

Reply:
removed